### ISSUE TITLE
[Tabstrip] accessibility issues

### ISSUE THREAD

**Issue by svetq** [telerik/kendo-angular#1583]
## I'm submitting a...

* Bug report 

## Current behavior

The active Tab element does not receive keyboard focus and screen reader user is not getting any information about the currently active tab and number of tabs in the Tablist

## Minimal reproduction of the problem with instructions

Check the functionality of the Tabstrip at the following article:
[https://www.telerik.com/kendo-angular-ui/components/layout/tabstrip/](https://www.telerik.com/kendo-angular-ui/components/layout/tabstrip/)

**Comment by Raisolution** [comment]
Closing this in favor of #2306. Fix will be available soon.
